composer.json

```diff
@@ -111,7 +96,6 @@
 	},
 	"require": {
 		"php": "^8.0|^8.1",
-		"block-editor-custom-alignments/block-editor-custom-alignments": "*",
```

Comment:
Curious why you're removing this (and the source block above)? I'm pretty sure we'll plan to use this on every project.